snapcraft.yaml

```diff
@@ -304,28 +304,30 @@ parts:
     build-environment: *buildenv
 
   harfbuzz:
-    after: [ fribidi ]
+    after: [ fribidi, meson-deps ]
     source: https://github.com/harfbuzz/harfbuzz.git
     source-tag: '5.3.1' # developers declared that they won't break ABI
     source-depth: 1
-    plugin: autotools
-    autotools-configure-parameters:
+    plugin: meson
+    meson-parameters:
       - --prefix=/usr
-      - --with-graphite2=yes
-      - --enable-introspection
-      - --with-gobject
-      - --enable-static
```

Comment:
I didn't realize we had enabled static before, I assume that's statically linked?  We probably didn't really want that, but with this change do you think we should be concerned we might break other builds?

Reply:
AFAIK, --enable-static doesn't link this statically, but just creates .a files to allow harfbuzz to be linked statically. I set "--default-library=both" in meson to build both dynamic and static libraries, so it will be possible to link it in both ways.

Reply:
Excellent, thanks for the clarification